documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -1,5 +1,6 @@
 [[writing-tests]]
 == Writing Tests
+:testDir: /media/data/home/rmannibucau/1_dev/junit5/documentation/src/test/java
```

Comment:
Please remove that. `{testDir}` is already defined in `documentation.gradle.kts`.

Reply:
oops, thanks for the catch